src/css/specific/notice-banner.css

```diff
@@ -4,10 +4,11 @@
   height: fit-content;
   margin-bottom: -20px;
   margin-top: -12px;
```

Comment:
you can take these margins out. They seem to cause the toolbar and the notice banner to overlap.

![Screenshot 2023-03-30 at 3 01 00 PM](https://user-images.githubusercontent.com/10934908/228973849-61f75f5f-15a3-4005-b270-c9cb290a0976.png)

Reply:
Fixed